<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="option_wd" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="load_img_pb">
           <property name="text">
            <string>Load Image</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="zoom_frm_wd">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="crop_pb">
              <property name="text">
               <string>Crop</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="resize_pb">
              <property name="text">
               <string>Resize</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="r_bg_pb">
              <property name="text">
               <string>Remove Background</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="fit_to_screen_pb">
              <property name="text">
               <string>Fit To Screen</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>       Zoom</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="zoom_in_pb">
              <property name="text">
               <string>IN</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="zoom_out_pb">
              <property name="text">
               <string>OUT</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="reset_zoom_pb">
              <property name="text">
               <string>Reset</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="remove_bg_wd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="white_bg_pb">
           <property name="text">
            <string>white</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="black_bg_pb">
           <property name="text">
            <string>black</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mask_red_pb">
           <property name="text">
            <string>mask_red</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mask_grey_pb">
           <property name="text">
            <string>mask_grey</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QPushButton" name="save_bg_pb">
              <property name="text">
               <string>Svae</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="img_wd" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="img_lay"/>
         </item>
         <item>
          <widget class="QWidget" name="img_option_wd" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QFrame" name="resize_wd">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QWidget" name="widget_6" native="true">
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="QWidget" name="widget_3" native="true">
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                     <item>
                      <widget class="QLabel" name="label_2">
                       <property name="minimumSize">
                        <size>
                         <width>50</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Width</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="width_le"/>
                     </item>
                     <item>
                      <widget class="QLabel" name="og_width">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="widget_4" native="true">
                    <layout class="QHBoxLayout" name="horizontalLayout_7">
                     <item>
                      <widget class="QLabel" name="label_3">
                       <property name="minimumSize">
                        <size>
                         <width>50</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Height</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="hight_le"/>
                     </item>
                     <item>
                      <widget class="QLabel" name="og_hight">
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget_5" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QPushButton" name="resize_apply_pb">
                    <property name="text">
                     <string>Apply</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="resize_undo_pb">
                    <property name="text">
                     <string>Undo</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="save_resize_pb">
                 <property name="text">
                  <string>Save</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="crop_pb_2" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QPushButton" name="apply_crop_pb">
              <property name="text">
               <string>Appy Crop</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
